src/templates/partials/traefik.hbs

```diff
@@ -0,0 +1,36 @@
+# generated with: {{{output.link}}}
+[entryPoints]
+[entryPoints.http.tls]
+{{#if (includes "modern" form.config)}}
```

Comment:
This is not the way to do this, it doesn't update as time goes along. The template has access to `output.protocols`, which is an array of the supported protocol versions. You can use the last element to get the minimum version, and then use the `replace` helper to change `TLSv1.2` to `VersionTLS12`, for example.